### DISCUSSION TITLE
PSSG Editor - Need Command Line Support - Export/Import dds file:  XXX_main.tga.dds

### DISCUSSION THREAD

**Discussion by Absolum7575** [Ryder25/Ego-Engine-Modding#73] [General]
Hi ! I want to try to make a Skin Generator (Livery Generator).

I tried to import only 1 file: evx_main.tga.dds
from games like:

Dirt
Dirt 2
Dirt 3
DiRT Rally
DiRT Showdown
Grid 2
Grid Autosport

in this game: 

T:\Games\Auto\Race Driver GRID ENG 1.3\cars\evx\livery_00\textures_high\evx_tex_high_00.pssg
T:Games\Auto\Race Driver GRID ENG 1.3\cars\evx\livery_00\textures_low\evx_tex_low_00.pssg

And the result is very good !

But I don't know how to - programming (c# , c++ , pyton and other).  
I only know how to use cmd commands and utilities on an advanced user.  
Plus I use ChatGPT help  ;)

Now I use gui program: Ego PSSG Editor 11.1.2017.0802
To import and export dds files.

But I really miss in this program - the support of the Command Line work.

At least it would be possible to Export and Import: evx_main.tga.dds file not 
via gui interface, but via command line.

Could you add such a function to the Ego PSSG Editor?
At least to be able to import and export 1 file: XXX_main.tga.dds 

If so.  I would love to hear from you at my email: mitwadspamo@gmail.com

p.s. Preferably of course the program should be x32 and x64.   But minimum at least x32
p.s.s. I need a binary program (a ready-made exe file)

____

p.s.s.s. And you didn't try to insert in any way (import) into the game:

Race Driver GRID ENG 1.3

Cars from other games ? 
So that the skin and physical characteristics were completely 
new car, which was not previously in the game Race Driver GRID ENG 1.3  ?

**Comment by ptasev** [comment]
Feel free to write yourself a command line app, but I'm not interested in doing so.
